tests-e2e/cypress/e2e/sdc-collection/sdcSchoolCollectionView.cy.ts

```diff
@@ -31,12 +31,6 @@ describe('SDC School Collection View', () => {
       //step three of collection - edit/verify data
       cy.get(selectors.studentLevelData.nextButton).should('exist').should('be.enabled').click();
 
-      // checking if the previous button is clickable and the user is taken to the previous step; brings to step 2
-      cy.get(selectors.studentLevelData.stepTwo).should('exist').click();
-
-      // Step three should be disabled
-      cy.get(selectors.studentLevelData.stepThree).should('exist').should('not.be.enabled');
-
```

Comment:
could we just comment these steps out instead of deleting them? I need to fix this flow anyhow.